.github/workflows/ci.yml

```
@@ -143,3 +143,33 @@ jobs:
         if-no-files-found: error
         retention-days: 14
 
+  ktlint:
+    runs-on: ubuntu-latest
+    steps:
+    # チェックアウト
+    - uses: actions/checkout@v2
+
+    # インストール
+    - run: |
+        curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.42.1/ktlint && chmod a+x ktlint && sudo mv ktlint /usr/local/bin/
```

[修正任意]
個人的には `&&` で繋げるより改行したほうが読みやすいです！

```suggestion
        curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.42.1/ktlint
        chmod a+x ktlint
        sudo mv ktlint /usr/local/bin/
```

---

自分はこだわりないけど、ktlintのドキュメントはワンライナーで書いてあります
https://github.com/pinterest/ktlint#installation

---

あ、なるほどです。
ただ今回は YAML で改行できるので、変えてもらえると嬉しいです 🙏